src/diff.js

```diff
@@ -29,6 +29,10 @@
  *
  * QUnit.diff( "the quick brown fox jumped over", "the quick fox jumps over" ) === "the  quick <del>brown </del> fox jump<ins>s</ins><del>ed</del over"
  */
+
+// Don't load QUnit.diff on non-Browser environments
+if ( typeof window !== "undefined" ) {
```

Comment:
I see. I guess with ES6 modules this problem could go away? If so, let's extend the comment a bit and leave the `if` as-is.